### PR TITLE
Hydroponics Tweaks

### PR DESCRIPTION
**Pull request by potato1234x**
## About the PR
Resprited hydroponics satchel, duffel bag, leather gloves, and ported tgstation seed extractor. Also fixed hydroponics bag name inconsistencies.

**Media**

prthumb

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Potato1234_x
- tweak: Changed sprites of the seed extractor, hydroponics bags and hydroponics leather gloves.
- fix: Fixed inconsistent hydroponics bag naming.

### CODE REVIEW

**Review by moonheart08 — CHANGES_REQUESTED** [review]
All RSIs need copyright updated.

**Comment by potato1234x** [comment]
the ones i changed are the only ones that needed changing, the others are still accurate

File: Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/2e24b7af2221928e4b844a29408e821b60a5fe29",
+  "license": "CC0-1.0",
```
Comment (crazybrain23) [inline Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json:3]: Since the original is ShareAlike, are you allowed to do this?

Comment (potato1234x) [reply Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json]: how did i mess that up

**Review by EmoGarbage404 — APPROVED** [review]

**Comment by potato1234x** [comment]
@moonheart08 its been a week, please review :(

File: Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/2e24b7af2221928e4b844a29408e821b60a5fe29",
+  "copyright": "Modified from tgstation at commit https://github.com/tgstation/tgstation/commit/547852588166c8e091b441e4e67169e156bb09c1",
```
Comment (AJCM-git) [inline Resources/Textures/Clothing/Back/Duffels/hydroponics.rsi/meta.json:4]: same for others
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/547852588166c8e091b441e4e67169e156bb09c1. Modified by potato1234x (github) for SS14",
```

**Review by AJCM-git — CHANGES_REQUESTED** [review]
merge master

**Comment by potato1234x** [comment]
i fucked it up i should just make a new pr

**Pull request closed without merging**